### PR TITLE
Update `/find`

### PR DESCRIPTION
**Pull request by juliasilge**
This PR updates the tables at `/find`. I added some new-ish packages 🎉 but I had to remove tidyposterior 😢 because its vignette title structure is not standard and `pkgdown::as_pkgdown()` fails.

### CODE REVIEW

**Comment by juliasilge** [comment]
Preview [for `/find` is here!](https://deploy-preview-244--tidymodels-org.netlify.app/find/)

File: content/find/all/index.Rmd [hunk 1]
```diff
@@ -23,11 +23,13 @@ Here are all the functions available across all of the [tidymodels packages](/pa
...
-pkgs <- c("baguette", "broom", "butcher", "corrr", "dials", "discrim", 
-          "embed", "hardhat", "infer", "modeldata", "modeldb", "parsnip", "plsmod",
+pkgs <- c("applicable", "baguette", "broom", "butcher", 
+          "corrr", "dials", "discrim", 
+          "embed", "finetune", "hardhat", "infer", "modeldata", "modeldb", 
+          "multilevelmod", "parsnip", "plsmod",
           "poissonreg", "probably", "recipes", "rsample", "spatialsample", 
-          "textrecipes", "tidyposterior", "tidypredict", "yardstick", 
-          "workflows")
+          "textrecipes", "tidypredict", "tune", "yardstick",
```
Comment (hfrick) [inline content/find/all/index.Rmd]: I would add `brulee`,`rules`,`shinymodels`,`stacks`, and `themis` here as well. And what about `vetiver`?

**Review by hfrick — APPROVED** [review]
Looks good, just a question about adding a few more packages to `/find/all`!

**Comment by juliasilge** [comment]
Ah, great catch! 🙌 Thank you

**Pull request merged**